REQUIRE

```diff
@@ -3,5 +3,6 @@ Compat 0.8.0
 Colors
 BinDeps 0.3.21
 Graphics 0.1
+BaseTestNext
```

Comment:
this should go in `test/REQUIRE`, not `REQUIRE` since it's only needed for tests